$NetBSD: patch-af,v 1.1 2009/08/26 21:44:43 he Exp $

Initialization of the math library version needs to be done
this way instead, ref. patch-ag

--- config/gen/platform/netbsd/misc.c.orig	2009-07-08 02:19:09.000000000 +0200
+++ config/gen/platform/netbsd/misc.c
@@ -26,17 +26,23 @@ Miscellaneous helper functions that are 
 =item C<void Parrot_platform_init_code(void)>
 
 Initialize Parrot for the NetBSD platform.
-So far only turns off SIGFPE for Alpha.
+So far turns off SIGFPE for Alpha, and
+ensures IEEE floating-point semantics from
+the math library.
 
 =cut
 
 */
 
 #include <signal.h>
+#include <math.h>
 
 void
 Parrot_platform_init_code(void)
 {
+
+    _LIB_VERSION = _IEEE_;	/* force IEEE math semantics and behaviour */
+
 #if defined(__alpha__)
     signal(SIGFPE, SIG_IGN);
 #endif
